enhancements/cluster-logging/cluster-logging-otel-support.md

```diff
@@ -0,0 +1,242 @@
+* In-cluster log storage for public, non-verified log ingestion.
+
+## Proposal
+
+In order to realize the goals of this enhancement:
+
+* Modify **cluster-logging-operator** to allow writing to **LokiStack** over ViaQ(the default) or OTLP based upon a **ClusterLogForwarder** annotation
+* Modify the vector log collector deployment to write to **LokiStack** over OTLP and additionally include select ViaQ attributes
+* Update the **loki-operator** to deploy v3.x are better of Loki to provided the an OTLP endpoint
+* Update the **loki-operator** to provide sane defaults to the new LokiStack OTEL API for server-side indexing of OTEL attributes to improve query performance
+* Modify the **logging-view-plugin** to use OTEL attributes that correspond to ViaQ *log_type*, *log_source*, *kubernetes.container_name*, *kubernetes.namespace_name*, *kubernetes.pod_name*
+
+### Workflow Description
+
+The following workflow describes deployment of a full logging stack to collect and forward logs to a Red Hat managed log store, **LokiStack**, using OTLP.
+
+**cluster administrator** is a human responsible for:
+
+* Managing and deploying day 2 operators
+* Managing and deploying an in-cluster LokiStack
+* Managing and deploying a cluster-wide log forwarder
+
+**cluster-observability-operator** is an operator responsible for:
+
+* managing and deploying observability operands (e.g. LokiStack, ClusterLogForwarder, Tracing) and console plugins (e.g console-logging-plugin)
+
+**loki-operator** is an operator responsible for managing a LokiStack instance.
+
+**cluster-logging-operator** is an operator responsible for managing log collection and forwarding.
+
+The cluster administrator does the following:
+
+1. Deploys the Red Hat **cluster-observability-operator**
+1. Deploys the Red Hat **loki-operator** which supports server-side indexing and OTLP ingestion endoint
+1. Deploys an instance of **LokiStack** in the `openshift-logging` namespace
+1. Deploys the Red Hat **cluster-logging-operator**
+1. Creates a **ClusterLogForwarder** custom resource for the **LokiStack**
+    1. **In Tech Preview, additionally annotated to instruct the operator to use the OTEL datamodel in lieu of ViaQ**
+
+The **cluster-observability-operator**:
+
+1. Deploys the console-logging-plugin for reading logs in the OpenShift console
+
+The **loki-operator**
+
+1. Deploys the **LokiStack** for storing logs in-cluster
+
+The **cluster-logging-operator**:
+
+1. Deploys the log collector to forward logs to log storage in the `openshift-logging` namespace
+
+
+### API Extensions
+
+During the time this feature is considered Tech Preview, instances of **ClusterLogForwarder** will be annotated to indicate to the
+operator that logs will be forwarded to **LokiStack** using OTLP:
+
+```
+   logstore.observability.openshift.io/otlp: [[LOKISTACK_NAMES]]
```

Comment:
I suggest to adapt this w/o reviving the `logstore` term from the old `ClusterLogging` resource:
```
   logging.observability.openshift.io/select-lokistack-otlp: [[LOKISTACK_NAMES]]
```
or
```
   observability.openshift.io/enable-lokistack-otlp: [[LOKISTACK_NAMES]]
```